rag/multimedia_rag/video_rag.py

```
@@ -0,0 +1,400 @@
+                self.video_table.add_computed_column(
+                    transcription=whisper.transcribe(
+                        audio=self.video_table.audio_track,
+                        model='base.en'
+                    )
+                )
+                
+        except Exception as e:
+            logger.error(f"Failed to create video table: {e}")
+            try:
+                self.video_table = pxt.get_table(self.table_name)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)